index.html

```diff
@@ -0,0 +1,238 @@
+                            <i class="fa fa-user"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                Forum
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Have the time to share your thoughts and opinions with experts for each topic.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-microphone"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                Workshop
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Try creating your own work using open source license rather than just watching at it.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-card">
+                        <div class="main-card-img">
+                            <i class="fa fa-cogs"></i>
+                        </div>
+                        <div class="main-card-title">
+                            <p>
+                                PANACCS Ignite
+                            </p>
+                        </div>
+                        <div class="main-card-description">
+                            <p>
+                                Get opportunities to network with PANACCS affiliates around the world, also after the summit.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <button type="button" class="main-btn">
+                Join PANACCS Global Summit 2022
+            </button>
+            <a href="https://google.com" class="main-link-btn">See the all program</a>
+        </div>
+    </main>
+    <section class="container speakers">
+        <div class="speaker-title">
+            <p>Featured Speakers</p>
+        </div>
+        <div class="speaker-container-card">
+        </div>
+        <button type="button" class="Feature_speaker_button">
+            MORE
+            <i class="fas fa-chevron-down"></i>
+        </button>
+    </section>
+    <footer>
+        <div class="partners">
+            <div class="partner-title">
+                <p>Partner</p>
+            </div>
+            <div class="container partner-group">
+                <p>Amazon</p>
+                <p>Google</p>
+                <p>Nvidia</p>
+                <p>CircleCI</p>
+                <p>Airbnb</p>
+            </div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure you use a `ul` tag as a container instead of a `div` tag for your partners' list and for each partner item a `<li>` tag instead of a `<p>` tag for best semantics tags.